src/Compilers/VisualBasic/Portable/Lowering/LambdaRewriter/LambdaFrame.vb

```diff
@@ -18,6 +19,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Private ReadOnly m_topLevelMethod As MethodSymbol
         Private ReadOnly m_sharedConstructor As MethodSymbol
         Private ReadOnly m_singletonCache As FieldSymbol
+        Friend ReadOnly ClosureOrdinal As Integer
 
         'NOTE: this does not include captured parent frame references 
         Friend ReadOnly m_captured_locals As New ArrayBuilder(Of LambdaCapturedVariable)
```

Comment:
Not part of this change but ... consider renaming 'm_captured_locals' to 'CapturedLocals' since it is declared Friend and consider making 'm_constructor' Private.